### PR TITLE
Fix Urls in MarkDown

### LINKED ISSUE #3441: [Docs] Feedback Documentation is missing the enabling and disabling of types of feedback
This url [feedback documentation](https://microsoft.github.io/botframework-solutions/virtual-assistant/handbook/feedback/) is extremely useful except at the part where it says:

FeedbackOptions consists of the following. You can alter any of the properties when you declare your options in startup.cs

Then it doesn't proceed to show an example of how to enable or disable any of the table of features below that line of text.  

I posted this to StackOverFlow two days ago and I received not a single response. The botframework documentation seems to do this a bunch.  Talk about how to do something like a new feature and documentation describe it, show parts of how it works, then leave out the most critical part of seeing the new functionality in C# code.

### LINKED ISSUE #3448: [Docs] Skill links are broken
/solution-accelerators/assistants/hospitality-assistant/

image

### PR DESCRIPTION
**Pull request by sidmore**
### Purpose
*What is the context of this pull request? Why is it being done?*
a. Updating Repo Urls for Skills
b. Show example of enabling or disabling certain properties of Feedback options

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [x] I have updated related documentation

### CODE REVIEW

**Review by pavolum — APPROVED** [review]

File: docs/_docs/virtual-assistant/handbook/feedback.md [hunk 1]
```diff
@@ -38,17 +38,38 @@ This implementation allows for feedback to be collected when a child dialog of m
...
 | FeedbackPromptMessage | Message to show after user when prompting for feedback | **string** | *Was that helpful?* |
 
+Example of altering any of the identified properties in startup.cs
+    ```csharp
+            services.AddSingleton(new FeedbackOptions
+            {
+                CommentsEnabled = false,
+                FeedbackActions =
+                {
+                    new Choice()
```
Comment (peterinnesmsft) [inline docs/_docs/virtual-assistant/handbook/feedback.md:48]: >() [](start = 30, length = 2)

nit: drop the parentheses - not necessary for parameterless constructors

File: docs/_docs/solution-accelerators/assistants/hospitality-assistant.md [hunk 2]
```diff
@@ -77,21 +77,21 @@ The majority of the skills connected to this sample are [experimental skills]({{
...
 1. Clone the following skills from our repository:
-    - [Hospitality Skill]({{site.repo}}/tree/master/skills/csharp/experimental/hospitalityskill)
-    - [Point of Interest Skill]({{site.repo}}/tree/master/skills/csharp/pointofinterestskill)
-    - [Weather Skill]({{site.repo}}/tree/master/skills/csharp/experimental/weatherskill)
-    - [Bing Search Skill]({{site.repo}}/tree/master/skills/csharp/experimental/bingsearchskill)
-    - [News Skill]({{site.repo}}/tree/master/skills/csharp/experimental/newsskill)
-    - [Restaurant Booking Skill]({{site.repo}}/tree/master/skills/csharp/experimental/restaurantbookingskill)
-1. [Deploy each one of these skills]({{site.baseurl}}/skills/tutorials/create-skill/csharp/4-provision-your-azure-resources/) separately, using the deployment script included in the skill directory. 
-1. [Add each skill]({{site.baseurl}}/skills/handbook/add-skills-to-a-virtual-assistant/) using the botskills connect CLI tool. 
+    - [Hospitality Skill](https://github.com/microsoft/botframework-skills/tree/master/skills/csharp/experimental/hospitalityskill)
```
Comment (peterinnesmsft) [inline docs/_docs/solution-accelerators/assistants/hospitality-assistant.md:87]: >https://github.com/microsoft/botframework-skills/ [](start = 26, length = 49)

I'm not clear how hard-coding the URIs like this fixes the underlying issue. What exactly was the problem to begin with? I tried looking at the original issue but I couldn't get a very clear understanding from the description there either. #Resolved

Comment (lauren-mills) [reply docs/_docs/solution-accelerators/assistants/hospitality-assistant.md:87]: The previous URLs were using this repo as the base URL, but after moving the skills to their own repo, it broke all the links.

**Review by lauren-mills — APPROVED** [review]

File: docs/_docs/solution-accelerators/assistants/hospitality-assistant.md (hunk 2 above)
Comment (peterinnesmsft) [reply docs/_docs/solution-accelerators/assistants/hospitality-assistant.md:87]: It turns out I can't read. My mind auto-completed as botframework-solutions :)

---
In reply to: [436982434](https://github.com/microsoft/botframework-solutions/pull/3453#discussion_r436982434) [](ancestors = 436982434)

**Review by peterinnesmsft — APPROVED** [review]

**Pull request merged**